.ycm_extra_conf.py

```diff
@@ -1,24 +1,18 @@
-import os
-import sys
-
+import sysconfig
 from pathlib import Path
 
-CONDA_PREFIX = os.environ["CONDA_PREFIX"]
+PREFIX = sysconfig.get_config_var("prefix")
 NVJITLINK_DIR = Path(__file__).parent
-VER = sys.version_info
-PYTHON_INCLUDE_NAME = "python3.10"  # Not ideal
 
-CONDA_INCLUDE_DIR = Path(CONDA_PREFIX, "include")
-PYTHON_INCLUDE_DIR = Path(CONDA_INCLUDE_DIR, PYTHON_INCLUDE_NAME)
+PREFIX_INCLUDE_DIR = Path(PREFIX, "include")
+PYTHON_INCLUDE_DIR = sysconfig.get_config_var("include")
 NVJITLINK_INCLUDE_DIR = Path(NVJITLINK_DIR, "include")
-CUDA_INCLUDE_DIR = "/usr/local/cuda-12.0/include"
```

Comment:
Figured search `PREFIX` would be good enough to pick these up

If not, we could add this back and use `/usr/local/cuda` here

Reply:
Missed this comment: https://github.com/rapidsai/pynvjitlink/pull/64#discussion_r1515235864

Added back in commit: https://github.com/rapidsai/pynvjitlink/pull/65/commits/1cfd7115ada953ed57c1592e39eaf3d2bafbca12